scripts/ios/helper.js

```diff
@@ -1,7 +1,6 @@
 var fs = require("fs");
 var path = require("path");
 var utilities = require("../lib/utilities");
-var xcode = require ("xcode");
```

Comment:
Hi, I've made these changes because there was a cordova 9 bug. I think this is reverting it to the original version of the plugin. 
https://github.com/RADAR-base/cordova-plugin-firebase/commit/0e5794288c693e477672617da457039d1ed547f7#diff-91f698c29d714cf5947fe5e7ae01dbaf